common/config/rush/pnpm-lock.yaml

```diff
@@ -8,7 +8,9 @@ dependencies:
   '@azure/ms-rest-js': 2.0.4
   '@azure/ms-rest-nodeauth': 0.9.3
   '@azure/storage-blob': 12.0.0-preview.2
-  '@microsoft/api-extractor': 7.3.8
+  '@microsoft/api-extractor': 7.3.11
```

Comment:
Was this file updated by running `rush update` or `rush update --full`?  It appears to be the latter, since unrelated dependencies are being updated.  Can you please revert this file and run `rush update` to just add/update the dependencies modified in this PR?